hnn_core/tests/test_network.py

```diff
@@ -93,7 +93,67 @@ def test_network():
                    'input_prox_A_weight_L5Pyr_ampa': 4.4e-5,
                    't0_input_prox': 50})
 
-    net = jones_2009_model(deepcopy(params), add_drives_from_params=True)
+    net = jones_2009_model(deepcopy(params), add_drives_from_params=False)
+
+    # add all drives explicitly and ensure that the expected number of drive
+    # cells get instantiated for each case
+    n_drive_cells_list = list()
+
+    weights_ampa_d1 = {'L2_basket': 0.006562,
+                       'L2_pyramidal': 7e-6,
+                       'L5_pyramidal': 0.142300}
+    weights_nmda_d1 = {'L2_basket': 0.019482,
+                       'L2_pyramidal': 0.004317,
+                       'L5_pyramidal': 0.080074}
+    synaptic_delays_d1 = {'L2_basket': 0.1,
+                          'L2_pyramidal': 0.1,
+                          'L5_pyramidal': 0.1}
+    n_drive_cells = 'n_cells'
+    n_drive_cells_list.append(n_drive_cells)
+    net.add_evoked_drive(
+        name='evdist1',
+        mu=63.53,
+        sigma=3.85,
+        numspikes=1,
+        location='distal',
+        n_drive_cells=n_drive_cells,
+        cell_specific=True,
+        weights_ampa=weights_ampa_d1,
+        weights_nmda=weights_nmda_d1,
+        synaptic_delays=synaptic_delays_d1,
+        seedcore=4)
+
+    weights_ampa_d2 = {'L2_basket': 0.006562,
+                       'L2_pyramidal': 7e-6,
+                       'L5_basket': 0.01,
+                       'L5_pyramidal': 0.142300}
+    weights_nmda_d2 = {'L2_basket': 0.019482,
+                       'L2_pyramidal': 0.004317,
+                       'L5_basket': 0.001,
+                       'L5_pyramidal': 0.080074}
+    synaptic_delays_d2 = {'L2_basket': 0.1,
+                          'L2_pyramidal': 0.1,
+                          'L5_basket': 0.1,
+                          'L5_pyramidal': 0.1}
```

Comment:
do you need these exact numbers for the following tests? If you're just adding a test for `n_cells` it probably doesn't matter what these numbers are

Reply:
Correct, the exact numbers don't matter here ( I just copied them from somewhere else). This is why we previously used `add_drives_from_params=True`.

Reply:
I guess I was trying to say you can just reuse the values from above if that's the case